src/commands/createStaticWebApp/addWorkspaceTelemetry.ts

```diff
@@ -0,0 +1,28 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as git from 'simple-git/promise';
+import { workspace } from 'vscode';
+import { IStaticWebAppWizardContext } from "./IStaticWebAppWizardContext";
+
+export async function addWorkspaceTelemetry(wizardContext: IStaticWebAppWizardContext): Promise<void> {
+    try {
+        wizardContext.telemetry.properties.isGitInstalled = 'true';
+
+        const localGit: git.SimpleGit = git(wizardContext.fsPath);
+        await localGit.status();
```

Comment:
We prioritize user experience over telemetry, so you should use `.then` instead of `await` to make sure this doesn't delay anything

Something like this: https://github.com/microsoft/vscode-azurefunctions/blob/2fd8a606a339439637f883dd1d2786635dece81d/src/funcCoreTools/getLocalFuncCoreToolsVersion.ts#L32

Reply:
Okay, FWIW this shouldn't take very long despite it being async since it's just a local `git status` command rather than an API call to a remote server.

Reply:
It's the principle of the thing. Protects us for cases that seem fast on our machines, but are actually noticeable on slower machines